CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.29.1] - 2021-05-31
+### Added
+- Prop `showBultAsUnit` to `quantity-selector` block.
+
```

Comment:
This is the correct format for a CHANGELOG entry:

```suggestion
## [Unreleased]
### Added
- Prop `showBultAsUnit` to `quantity-selector` block.

```

Reply:
Also, could you update the description so that it matches the name of the prop after you've changed it?